src/styles/layout.sss

```diff
@@ -94,16 +94,16 @@
     overflow: auto
     padding: 1em
 
-#exitcode
-    padding-top: 1em
-    color: #0000007f
-
 #errmsg
-    color: #d00a0a
+    color: red
```

Comment:
Please keep using the existing colors. They were chosen explicitly.

Reply:
I really think more bright red is better for error messaging especially considering our output will now have colors.
Previous color was somehow brownish and kinda not bright enough to highlight error.

Why such color was choosen?